org.kde.ruqola.json

```diff
@@ -0,0 +1,55 @@
+{
+    "id": "org.kde.ruqola",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "ruqola",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host",
```

Comment:
that's a lot of permissions for a chat client.

Reply:
Yes, what do you need to read or write that isn't handled by the portals already?